src/gei/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -464,6 +483,11 @@ private void ValidateGHESOptions(MigrateRepoCommandArgs args, bool cloudCredenti
                 {
                     throw new OctoshiftCliException("Either --aws-secret-key or AWS_SECRET_KEY environment variable must be set.");
                 }
+
+                if (!GetAwsRegion(args).HasValue())
+                {
+                    _log.LogWarning("Please consider providing --aws-region or AWS_REGION environment variable. It will be required in future releases.");
```

Comment:
> Specifying an AWS region with the --aws-region argument or AWS_REGION environment variable is currently not required, but will be required in a future release. Defaulting to us-east-1.

Reply:
Addressed in 98de164b93e6c5ad0a9f24c52673cd80b4f9a7fc